app/views/dashboard/_status_chart_widget.html.erb

```diff
@@ -10,9 +10,9 @@
     <%= title = _('Host Configuration Chart') %>
   <% end %>
 </h4>
-<div id="<%= widget_chart_id %>" class="host-configuration-chart">
+<div class="host-configuration-chart">
```

Comment:
I left the class here because `DonutChart` is mounted in other places (i.e  [openscap plugin](https://github.com/theforeman/foreman_openscap/blob/b4ea33b3840b78f6034312820f73ecc7d17683cd/app/views/compliance_hosts/show.html.erb#L29)) therefore the class is too specific to the widget alignment and there is no dynamic class support in the component.